src/App.jsx

```diff
@@ -14,33 +15,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const prepareGoods = (goods, { field, isReverse }) => {
+  const sortedGoods = [...goods];
```

Comment:
There is a typo in the array spread syntax. It should be `const sortedGoods = [...goods];` instead of `const sortedGoods = [.goods];`.